docs/modules/ic-identity-guide/pages/auth-how-to.adoc

```diff
@@ -6,21 +6,23 @@
+The Internet Identity enables you to authenticate securely and anonymously when you access applications that use it as an authentication method. A different identity is created for each application you log in to, and you will be able to use all of your registered devices or authentication methods to log in to the same account. This is all accomplished because Internet Identity is built to run on the {IC} blockchain, ensuring cryptographic security, and leverages the browser standard, Web Authentication (WebAuthn). 
 
-Unlike most authentication methods, your Internet Identity does not require you to set and manage passwords or provide any personal identifying information to applications or to Internet Identity. Instead, you create authentication profiles using an ID number and the authentication methods you choose such as facial recognition from a smartphone, your computer unlock password, or a security key. 
+Unlike most authentication methods, your Internet Identity does not require you to set and manage passwords or provide any personal identifying information to applications or to Internet Identity. Instead, you register a device by using any of the device's own authentication methods. If the device supports WebAuthn, and accepts your authentication method, a public key is created and is represented by an ID number (you User Number). At the same time, the security chip in your device generates a  private key, which creates a signature associated with your ID number. The private key is secret and never leaves your device. No one, not even you, can see the private key. The unique signature from your private key is what Internet Identity uses to prove that you are in possession of the correct private key for your ID number. 
 
-NOTE: On macOS/iOS/ipadOS (including iPhones), authentication using Safari is coupled to your browser profile. If you want to log in to an application in a different browser, or if you use multiple Safari browser profiles, you have to add the combination of your authentication method and the new browser as a new device. See: <<Add a device,`+Add a device+`>>.
+When you access an application that uses Internet Identity, you will be re-directed to the Internet Identity app. Once you authenticate into the app, Internet Identity creates a unique identity that is saved in the application's persistance layer (or web server), but does not contain any personal information, because at no point have you offered any personal information. However, this unique and anonymous identity is associated with your public ID number, so if you lose the number, you will also lose the account identity associated with it.
```

Comment:
I don't understand what you mean by "that is saved in the application's persistence layer" but we will definitely want to drop the term "web server" there.

Reply:
So where would the Client Id created for the user live? In an asset canister? Also, I thought that II could work on regular web apps as well as the IC